src/pages/DatastudioDashboard/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+    width: 100%;
+    margin-top: -2.5rem;
+
+    @media only screen and (max-width: 768px) {
+        margin-top: -2rem;
+    }
+
+    @media only screen and (max-width: 480px) {
+        margin-top: -1rem;
+    }
+`
+
+export default function DatastudioDashboard(): JSX.Element {
+    return (
+        <Wrapper>
+            <iframe
+                title="Datastudio Dashboard"
+                width="100%"
+                height="2000px"
```

Comment:
why 2000?

Reply:
Perfect value is 9**px, i think this was left will update it as 2000px doesn't change anything on styling